keyboards/yd68/xm04/main.c

```diff
@@ -0,0 +1,99 @@
+#include <avr/io.h>
+#include <avr/power.h>
+#include <avr/wdt.h>
+#include "lufa.h"
+#include "print.h"
+#include "sendchar.h"
+#include "xm04.h"
+#include "xm04_task.h"
+//#include "serial.h"
+#include "keyboard.h"
+#include "keycode.h"
+#include "action.h"
+#include "action_util.h"
+#include "wait.h"
+#include "suart.h"
+#include "suspend.h"
+extern bool dozing;
+extern bool sleeping;
+extern bool force_usb;
+
+static int8_t sendchar_func(uint8_t c)
```

Comment:
This is a statically defined routine that is only available in this `.c` file, but is not used. You can remove this.

Reply:
Will removing this and the reference to it below still allow debug output on the hid_listen console, if needed? It's something related to that, I know, but I'm not quite sure how all of it's tied together.